src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources.java

```diff
@@ -595,7 +595,7 @@ public class XPATHErrorResources extends ListResourceBundle
   { ER_RTF_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
        "rtf() not supported by XRTreeFragSelectWrapper"},
 
-  { ER_RTF_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
+  { ER_ASNODEITERATOR_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
```

Comment:
Please replace the usage in XRTreeFragSelectWrapper, and also update the LastModified tag.